src/components/molecules/Common/AssetModal/AssetCard/index.tsx

```diff
@@ -35,8 +33,12 @@ const AssetCard: React.FC<Props> = ({
       selected={selected}
       cardSize={cardSize}
       onClick={() => onCheck?.(!check)}>
-      <ImgWrapper cardSize={cardSize} url={url}>
-        {!isImage && <Icon icon="file" />}
+      <ImgWrapper cardSize={cardSize}>
+        {/\.(jpg|jpeg|png|gif|svg|webp|GIF|JPG|JEPG|PNG|SVG|WEBP)$/.test(url) ? (
+          <PreviewImage url={url}></PreviewImage>
```

Comment:
Unless there is a JEPG image format I am unaware of that is a typo ;) Should be JPEG, right?

Reply:
yesyes, sorry, I just copy from inoue-san, thanks

Reply:
Also if PreviewImage is getting no children, just make it
`<PreviewImage url={url} />`